packages/integration-tests/src/entities/AuthorCodegen.ts

```diff
@@ -108,6 +110,7 @@ export interface AuthorOpts {
   favoriteShape?: FavoriteShape | null;
   mentor?: Author | AuthorId | null;
   currentDraftBook?: Book | BookId | null;
+  favoriteBook?: Book | BookId | null;
```

Comment:
Suggestion: Let's update `generateEntityCodegenFile` to not output this since users don't need to pass it in.